### PR TITLE
Update changelog for 8.7 release

### PR DESCRIPTION
**Pull request by lahsivjar**
Updated changelogs for 8.7 release.

### CODE REVIEW

**Comment by lahsivjar** [comment]
I have combined entries for similar features on different aggregators in a single entry as well as moved some of the changelog entries so that the numerous changes to the aggregations are together. Let me know if others have any concerns/thoughts about this.

File: changelogs/8.7.asciidoc [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+- Add a service summary metrics aggregator that writes to `metrics-apm.service_summary.${interval}` with `1m`, `10m` and `60m` intervals {pull}10029[10029]
+- Service metrics and service summary aggregations are now always enabled {pull}10060[10060] {pull}10109[10109]
+- Dedicated overflow buckets for transaction, service, service destination and service summary aggregation to limit cardinality {pull}10069[10069] {pull}9856[9856] {pull}10061[10061]
+- Automatically scale `MaxGroups`, `MaxTransactionGroups` and `MaxServices` based on available memory {pull}9856[9856] {pull}10157[10157]
+- Set data streams with non-default rollup intervals as hidden {pull}10070[10070]
+- Rename service metrics to service transaction metrics {pull}10095[10095]
+- Reduce memory usage of transaction and service transaction aggregations {pull}10178[10178]
+- Metrics data streams now use synthetic source {pull}9756[9756]
+- Translate otel log events to error events {pull}10066[10066]
+- Use Elasticsearch fetcher for agent remote configuration where possible {pull}9720[9720] {pull}10088[10088]
```
Comment (axw) [inline changelogs/8.7.asciidoc:51]: @lahsivjar could you please add a changelog entry for https://github.com/elastic/apm-server/pull/9722?

@kruskall please try to remember to add changelog entries for major changes like this in the future

**Review by axw — APPROVED** [review]

**Review by bmorelli25 — CHANGES_REQUESTED** [review]
Can you please add `* <<release-notes-8.7>>` above this line:

https://github.com/elastic/apm-server/blame/3a12640024314ef3aa313708fcd34702f79ca6ba/docs/release-notes.asciidoc#L12

File: changelogs/8.7.asciidoc [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+[float]
+==== Breaking Changes
+- `transaction.failure_count` has been removed. `transaction.success_count` type has changed to `aggregated_metric_double` {pull}9791[9791]
+- `transaction.success_count` has been moved to `event.success_count` {pull}9819[9819]
+- Stop indexing transaction metrics to `metrics-apm.internal` {pull}9846[9846]
+- Stop indexing span destination metrics to `metrics-apm.internal` {pull}9926[9926]
+
+[float]
+==== Deprecations
```
Comment (bmorelli25) [inline changelogs/8.7.asciidoc:20]: Let's remove this heading since there aren't any deprecations

```suggestion
```

**Comment by lahsivjar** [comment]
@bmorelli25 I have updated the PR with the suggested changes. Merging this, let me know if there are any other changes, I will create a separate PR.

**Pull request merged**